RedHatInsights/insights-rbac-ui
# [DO NOT MERGE]Pf dependencies debugging

This PR shows that for some reason, referencing pf-react-table causes the whole PF react-core to be injected into the production build.

How to reproduce:

1. install dependencies
2. run `yarn analyze`, you will see a graph in browser, the whole bundle will have 784.2 KB
3. go here and use PF react-table import instead of const override: https://github.com/RedHatInsights/insights-rbac-ui/pull/651/files#diff-0f4d89b4ef9387451ee47bbf2648ab8a4eba081195d459390f68842902738ea4R31
4. run `yarn analyze`, you will see a graph in browser, the whole bundle will have 2.05 MB and all PF components are in the bundle. Search for about modal or login page. It will be present in the graph even though they are never used in rbac UI.

cc @RedHatInsights/platform-experience @karelhala @redallen

@evwilkin I know that you are aware I just wanted to show a real example as well.

### Witouth PF
![screenshot-127 0 0 1_8888-2021 01 22-14_47_55](https://user-images.githubusercontent.com/22619452/105498742-d57ac080-5cc0-11eb-8b12-82b5a5698cf9.png)

### With PF
![screenshot-127 0 0 1_8888-2021 01 22-14_48_17](https://user-images.githubusercontent.com/22619452/105498823-ea575400-5cc0-11eb-8a76-1107991aba38.png)
